java/libphonenumber/pom.xml

```diff
@@ -30,6 +30,26 @@
     </testResources>
     <plugins>
       <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.2.0</version>
+        <configuration>
+          <instructions>
+            <Export-Package>com.google.i18n.phonenumbers</Export-Package>
+            <Private-Package>com.google.i18n.phonenumbers.data</Private-Package>
```

Comment:
Based on my understanding of OSGi, these directives only have an impact on how Classes are exposed in an OSGi environment. Since there are only resources within the data directory this statement would actually have no impact on the generated meta-data. I'm going to remove it since it has no actual effect.